From f9fa8e515d906ee4e66c89a3d6fa728e22579ba5 Mon Sep 17 00:00:00 2001
From: ted malaska <ted.malaska@cloudera.com>
Date: Mon, 12 Oct 2015 11:23:16 -0400
Subject: [PATCH 201/217] HBASE-14158 Add documentation for Initial Release
 for HBase-Spark Module integration

Signed-off-by: Misty Stanley-Jones <mstanleyjones@cloudera.com>

Conflicts:
	src/main/asciidoc/_chapters/spark.adoc

Change-Id: I5dd65ba6784778f0bdecbbd0326acc4297e54028
Reason: New Feature
Author: Ted Malaska
Ref: CDH-35578
---
 src/main/asciidoc/_chapters/spark.adoc |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/main/asciidoc/_chapters/spark.adoc b/src/main/asciidoc/_chapters/spark.adoc
index 37503e9..4f8b00b 100644
--- a/src/main/asciidoc/_chapters/spark.adoc
+++ b/src/main/asciidoc/_chapters/spark.adoc
@@ -36,9 +36,9 @@ more information on the Spark project and subprojects. This document will focus
 on 4 main interaction points between Spark and HBase. Those interaction points are:
 
 Basic Spark::
-  The ability to have an HBase Connection at any point in your Spark DAG.
+  The ability to have a HBase Connection at any point in your Spark DAG.
 Spark Streaming::
-  The ability to have an HBase Connection at any point in your Spark Streaming
+  The ability to have a HBase Connection at any point in your Spark Streaming
   application.
 Spark Bulk Load::
   The ability to write directly to HBase HFiles for bulk insertion into HBase
@@ -205,7 +205,7 @@ There are three inputs to the `hbaseBulkPut` function.
 . The hbaseContext that carries the configuration boardcast information link us
 to the HBase Connections in the executors
 . The table name of the table we are putting data into
-. A function that will convert a record in the DStream into an HBase Put object.
+. A function that will convert a record in the DStream into a HBase Put object.
 ====
 
 == Bulk Load
-- 
1.7.9.5

